src/components/HtmlView.js

```diff
@@ -31,34 +29,32 @@ const cssRules =
   font-size: ${normalize(13)}px;
 }
 th {
-  border-bottom: 0.25px solid ${'#3f5c7a'};
-  border-left: 0.25px solid ${'#3f5c7a'};
-  border-top: 0.25px solid ${'#3f5c7a'};
+  border-bottom: 0.68px solid ${'#3f5c7a'};
```

Comment:
why exactly `0.68px`? how did you find out?

Reply:
i know you won't be happy with my answer. i found it by trying.
if we give a value below 0.68px, the borders do not appear. there is nothing about this in the document.

Reply:
|0.25px|0.68px|1px|
|--|--|--|
![Simulator Screen Shot - iPhone 8 Plus - 2023-03-02 at 16 21 09](https://user-images.githubusercontent.com/11755668/222471210-f2268237-8a53-4b97-8f08-20314e26f5cd.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2023-03-02 at 16 21 14](https://user-images.githubusercontent.com/11755668/222471275-45cee6d4-3988-42b5-9547-6638ef1286f0.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2023-03-02 at 16 21 26](https://user-images.githubusercontent.com/11755668/222471292-d5d89ac2-3f78-4265-9c1a-2f8c1149962c.png)

Reply:
all right, let's take 1px i suggest. looks fine.

Reply:
ok then I update the code according to 1px. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/fed00e32327acd9474e3c571abf33a7a4fee0c04